gui/src/components/settings/pages/components/StayAlignedSettings.tsx

```diff
@@ -77,6 +78,68 @@ export function deserializeStayAlignedSettings(
   return serialized;
 }
 
+function CopySettingsButton({ values }: { values: SettingsForm }) {
+  const { l10n } = useLocalization();
+  const { currentLocales } = useLocaleConfig();
+  const degreeFormat = new Intl.NumberFormat(currentLocales, {
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+  });
+
+  const trackers = useAtomValue(connectedIMUTrackersAtom);
+
+  function boolify(value: boolean) {
+    return value ? 'true' : 'false';
+  }
+
+  const copySettings = () => {
+    const config = values.stayAligned;
+
+    let debug = 'Stay Aligned\n';
```

Comment:
use ``` ` ``` syntax

you will be able to do

debug = `
  Stay Aligned
  
  GENERAL
  =======
  
`;

easier to maintain